tests/format/asciiB.dhall

```diff
@@ -0,0 +1 @@
+\(a : Type) -> forall (b : a) -> a
```

Comment:
Does this test work with a missing newline at the end of the file?

Reply:
No.  We have to strip trailing newlines because it looks for an exact match with the rendered string (which doesn't have a trailing newline)

Reply:
@Gabriel439 Would it be ok to change the test matcher to append a newline to the rendered string? Or, equivalently, to strip the last character from the loaded file? It would be nice for the test files to have the usual final newline.